.github/workflows/e2e-matrix.yml

```diff
@@ -36,8 +37,8 @@ jobs:
             node_version: 22
           # - os: macos-latest
           #   node_version: 23
-          - os: windows-latest
-            node_version: 22
+          # - os: windows-latest
```

Comment:
It'd be great to fix up the Gradle issue in this PR, but if not there should be a `TODO` comment and a fast-follow on it so we're not missing Windows in nightlies. @xiongemi can help out

Reply:
Added TODO also assigned a task for @xiongemi to follow-up